### PR TITLE
[5.4] use same output style for jobs as migrations

### PR DESCRIPTION
**Pull request by Quezler**
- Adds a message for when a job has started.
- Make status strings the same width.

See 2328c7a
And laravel/framework#18542

### CODE REVIEW

**Review by m1guelpf — APPROVED** [review]

**Review by CedriQ — APPROVED** [review]
Slick

**Comment by tillkruss** [comment]
Would you mind adding tests for this?

**Comment by Quezler** [comment]
@tillkruss If you point me in the direction of the existing tests for job output, sure 👍

**Comment by Quezler** [comment]
And i currently do not see how a test would be written for this/wether it is required at all (the writeOutput function only gets called in the function i changed, from all the files in the entire framework)

File: src/Illuminate/Queue/Console/WorkCommand.php [hunk 1]
```diff
@@ -137,15 +142,21 @@ protected function listenForEvents()
...
+        switch ($status) {
+            case 'starting':
+                $this->output->writeln('<comment>['.Carbon::now()->format('Y-m-d H:i:s').'] Processing:</comment> '.$job->resolveName());
+                break;
+            case 'success':
+                $this->output->writeln('<info>['.Carbon::now()->format('Y-m-d H:i:s').'] Processed:</info>  '.$job->resolveName());
+                break;
+            case 'failed':
+                $this->output->writeln('<error>['.Carbon::now()->format('Y-m-d H:i:s').'] Failed:</error>     '.$job->resolveName());
+                break;
```
Comment (tillkruss) [inline src/Illuminate/Queue/Console/WorkCommand.php]: Using `str_pad()` here, might make this more readable.

**Comment by Quezler** [comment]
I'll refractor it in a moment, good point @tillkruss

**Comment by Quezler** [comment]
@tillkruss I have updated the code the best i could, is it more readable now?

**Pull request merged**